### ISSUE TITLE
An Example is needed for plain WMS

### ISSUE THREAD

**Issue by devdattaT** [kartena/Proj4Leaflet#51]
I have a simple WMS in EPSG:32643 (UTM 43N), which is served by GeoServer. I'm trying to use Leaflet (with and without this plugin) and am failing. The map does not show my data, and no request is made to the service. This WMS is not tiled.
- When I use this Plugin, I don't know what parameters to provide for  `origins`. 
- I know that instead of the origins, we can use the Transformation parameter, but I could not figure out what 4 parameters to pass to the `L.Transformation()` constructor.

Technically it should be possible to use non-tiled WMS services, since I could successfully use  a simple WMS Service in EPSG:4326, by  using the  the following crs constructor:

```
var crs = new L.Proj.CRS('EPSG:4326',
        '+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs',
    {
        transformation:new L.Transformation(1 / 360, 0.5, -1 / 360, 0.5),
        resolutions: [
             0.703125, 0.3515625, 0.17578125, 0.087890625, 0.0439453125, 0.02197265625, 
             0.010986328125,  0.0054931640625, 0.00274658203125, 0.001373291015625
        ]
    });
```

So to summarize, it would be helpful to provide a sample which shows how to use the plugin when one does not have a tiling scheme; or how to select the parameters for the `L.Transformation()` constructor.
